### ISSUE TITLE
testing

### ISSUE THREAD

**Issue by sanjanasri** [neubig/lamtram#20]
I used this command 
"src/lamtram/lamtram-train --dynet_mem 1024 --model_type encdec --layer_size 6 --trainer adam --learning_rate 0.001 --seed 0 --train_src train-src.unk --train_trg train-trg.unk --dev_src dev-src.txt  --dev_trg dev-trg.txt --model_out transmodel.out" for training the translation model ans used below for testing

"src/lamtram/lamtram --dynet_mem 1024 --operation ppl --src_in test_src.txt --models_in encdec=transmodel.out > results.txt" 

But, I am not getting any output. Please help

**Comment by geovedi** [comment]
i assume you want to check the ppl of the translated text, use the following command

    src/lamtram/lamtram --dynet_mem 1024 --operation ppl --src_in test_src.txt --models_in encdec=transmodel.out < results.txt

but if you want to generate the translation, use the following

    src/lamtram/lamtram --dynet_mem 1024 --operation gen --src_in test_src.txt --models_in encdec=transmodel.out > results.txt

read more here, https://github.com/neubig/nmt-tips
